### ISSUE TITLE
Shipping Flow definitions without declaraa

### ISSUE THREAD

**Issue by mAAdhaTTah** [stylelint/stylelint#2398]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug

> Which version of stylelint are you using?

7.9.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

WebPack Stylelint Plugin

> What did you expect to happen?

Flow should not throw errors for Flow types shipped in `node_modules`.

> What actually happened (e.g. what warnings or errors you are getting)?

Because stylelint ships Flow definitions inline with the code, Flow checks those files for type validity, but because the `decls` folder is missing, the consuming project cannot add the declarations used by stylelint, so Flow returns a lot of errors like these:

```flow
node_modules/stylelint/lib/utils/isStandardSyntaxAtRule.js:10
 10: module.exports = function (atRule/*: postcss$atRule*/)/*: boolean*/ {
                                          ^^^^^^^^^^^^^^ identifier `postcss$atRule`. Could not resolve name
```

> Suggestion

Strip the flow definitions in the `npm` module or ship the `decls` folder to `npm` so the consumer can add them to their flow configuration.

**Comment by hudochenkov** [comment]
Thank you for reporting! This problem was already reported. Check #2322
